tests/ballerina-unit-test/src/test/resources/test-src/reflect/equal.bal

```diff
@@ -166,6 +166,34 @@ function testFloatFalsePositive() returns (boolean) {
 
 // End Float
 
+// Start Decimal
+
+function testDecimalTruePositive() returns (boolean) {
+    decimal d1 = 2.12;
+    decimal d2 = 2.12;
+    return reflect:equals(d1,d2);
+}
+
+function testDecimalTrueNegative() returns (boolean) {
+    decimal d1 = 2.12;
+    decimal d2 = 5.12;
+    return reflect:equals(d1,d2);
+}
+
+function testDecimalFalseNegative() returns (boolean) {
+    decimal d1 = 2.12;
+    decimal d2 = 2.12;
+    return !reflect:equals(d1,d2);
+}
+
+function testDecimalFalsePositive() returns (boolean) {
```

Comment:
```suggestion
function testDecimalFalsePositive() returns boolean {
```

Shall we remove the parentheses around the return type in all the new additions, since they are not tuple types?

Reply:
+1. Will remove the parentheses as suggested.